src/Mondu/Mondu/MonduRequestWrapper.php

```diff
@@ -296,6 +289,72 @@ public function order_refunded($order_id, $refund_id) {
    * @throws ResponseException
    */
   public function cancel_invoice($mondu_order_id, $mondu_invoice_id) {
-    $this->api->cancel_invoice($mondu_order_id, $mondu_invoice_id);
+    $this->wrap_with_mondu_log_event('cancel_invoice', array($mondu_order_id, $mondu_invoice_id));
+  }
+
+  /**
+   * @param $topic
+   *
+   * @throws MonduException
+   * @throws ResponseException
+   */
+  public function register_webhook(string $topic) {
+    $response = $this->wrap_with_mondu_log_event('register_webhook', array($topic));
+
+    return $response['webhooks'];
+  }
+
+  /**
+   * @throws MonduException
+   * @throws ResponseException
+   */
+  public function get_webhooks() {
+    $response = $this->wrap_with_mondu_log_event('get_webhooks');
+
+    return $response['webhooks'];
+  }
+
+  /**
+   * @throws MonduException
+   * @throws ResponseException
+   */
+  public function webhook_secret() {
+    $response = $this->wrap_with_mondu_log_event('webhook_secret');
+
+    return $response['webhook_secret'];
+  }
+
+  /**
+   * @param $exception
+   *
+   * @throws MonduException
+   * @throws ResponseException
+   */
+  public function log_plugin_event(\Exception $exception, string $event, $body = null) {
+    global $wp_version;
+    $params = [
+      'plugin' => 'WOOCOMMERCE',
+      'version' => MONDU_PLUGIN_VERSION,
+      'language_version' => phpversion(),
```

Comment:
Maybe we should add the PHP prefix here like in the spec/contracts/plugin_events/log_contract_spec.rb
( so it will be 'language_version' => 'PHP '.phpversion(),